src/porepy/grids/simplex.py

```diff
@@ -82,7 +82,9 @@ def __init__(
 
         # Face node relations. Each face is oriented from low to high node index.
         face_nodes.sort(axis=1)
-        face_nodes, _, cell_faces = pp.array_operations.unique_rows(face_nodes)
+        face_nodes, _, cell_faces = np.unique(
```

Comment:
We can avoid having `_` among return values by not passing `return_index=True` here (and probably at some other places)